packages/database/src/testUtilities/upsertDummyRecord.js

```diff
@@ -24,13 +24,15 @@ const CUSTOM_DUMMY_VALUES = {
   },
 };
 
-const processDefaultValue = defaultValue => {
+const processDefaultValue = (defaultValue, columnType) => {
   const [preColonValue, type] = defaultValue.split('::');
   const value = preColonValue.replace(/^'/, '').replace(/'$/, '');
+  const dataType = type || columnType;
 
   // handle exceptional types
-  switch (type) {
+  switch (dataType) {
     case 'jsonb':
+    case 'boolean':
```

Comment:
Boolean default values are returned as strings from the database (eg. `'false'` instead of `false`)